src/components/Forms/UserAccount/index.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import Box from "@material-ui/core/Box";
+import Card from "@material-ui/core/Card";
+import Typography from "@material-ui/core/Typography";
+import useStyles from "./styles";
+
+const UserAccount = () => {
+  const classes = useStyles();
+
+  return (
+    <Card className={classes.card}>
+      <Box style={{ display: "flex", alignItems: "center", marginBottom: 10 }}>
+        <Typography className={classes.text} style={{ width: "56%" }}>
```

Comment:
styles can be moved into styles file as there are multiple places that use the same style(reusability)